CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## v1.5.0 / 2021-08-20
+
+* [#381](https://github.com/fsnotify/fsnotify/pull/381) Go: Increase minimum required version to Go 1.12
+* [#289](https://github.com/fsnotify/fsnotify/pull/298) Feature: Add AddRaw method which does not follow symlinks when adding a watch
+* [#289](https://github.com/fsnotify/fsnotify/pull/289) Windows: Follow symlinks by default like on all other systems
+* [#378](https://github.com/fsnotify/fsnotify/pull/378)
+   [#381](https://github.com/fsnotify/fsnotify/pull/381)
+   [#385](https://github.com/fsnotify/fsnotify/pull/385)
+   CI: Use GitHub Actions for CI and cover go 1.12-1.17
+* [#325](https://github.com/fsnotify/fsnotify/pull/325) Go 1.14+: Fix unsafe pointer conversion
```

Comment:
Thanks for writing this up

I think it would be preferable to put the issue/PR # at the end, just to follow the format used previously (see 1.4.1, etc.)